### PR TITLE
Added the case for reading blank cells as empty strings

### PR DESCRIPTION
**Pull request by Nikitapaliwal123**
Added the case for reading blank cells as empty strings in place of 'NULL' strings.

### CODE REVIEW

File: core-plugins/src/main/java/io/cdap/plugin/batch/source/ExcelInputReader.java [hunk 1]
```diff
@@ -90,7 +90,7 @@ public class ExcelInputReader extends BatchSource<LongWritable, Object, Structur
   private static final String RECORD = "record";
   private static final String EXIT_ON_ERROR = "Exit on error";
   private static final String WRITE_ERROR_DATASET = "Write to error dataset";
-  private static final String NULL = "NULL";
+  private static final String NULL = "";
```
Comment (MrRahulSharma) [inline core-plugins/src/main/java/io/cdap/plugin/batch/source/ExcelInputReader.java:93]: Please add a Unit Test to validate the null scenario.

**Comment by MrRahulSharma** [comment]
Excel file change is not required in this PR.

File: core-plugins/src/main/java/io/cdap/plugin/batch/source/ExcelInputReader.java (hunk 1 above)
Comment (MrRahulSharma) [reply core-plugins/src/main/java/io/cdap/plugin/batch/source/ExcelInputReader.java:93]: Discussed offline the UT for this does not run today, hence no change required here.
